### PR TITLE
fix(images): fixed update-drivers-website image build and run.

### PR DESCRIPTION
**Pull request by FedeDP**
I am still unsure about whether we will have proper `aws` secrets to push on s3 @maxgio92

### CODE REVIEW

**Comment by FedeDP** [comment]
Ok, i think we need to add:

```
serviceAccountName: driver-kit
```
to the job config file.
See: https://github.com/falcosecurity/test-infra/blob/master/config/prow/build-drivers-serviceaccount.yaml

File: images/update-drivers-website/entrypoint.sh [hunk 1]
```diff
@@ -31,14 +31,14 @@ function check_program {
...
+    check_program "aws"
     check_program "update-drivers-website"
     
     # Build updated json
-    update-drivers-website /tmp/website
+    update-drivers-website images/update-drivers-website/
     
     # Push the updated files to s3 bucket
-    aws s3 cp "/tmp/website/" s3://${S3_DRIVERS_BUCKET}/${S3_DRIVERS_KEY_PREFIX}/site --recursive --include "*.json" --include "index.html" --acl public-read
+    aws s3 cp "images/update-drivers-website/" s3://${S3_DRIVERS_BUCKET}/${S3_DRIVERS_KEY_PREFIX}/site --recursive --include "*.json" --include "index.html" --acl public-read
```
Comment (maxgio92) [inline images/update-drivers-website/entrypoint.sh:41]: Instead, why not shipping the index.html artifact with the image, copying it from that and avoid decorating the prowjob with the repostiory? (if the index.html artifact is supposed to be versioned).

In any case, if it makes sense to you @FedeDP, is only an improvement.

**Review by maxgio92 — APPROVED** [review]
/approve

**Pull request merged**